tests/unit/logging/test_logging.py

```diff
@@ -37,6 +38,23 @@ def test_logging_default():
             assert len(logger.handlers) == 2
 
 
+def test_logging_level():
+    with JinaLogger('test_logger', log_config=os.path.join(cur_dir, 'yaml/file.yml')) as logger:
+        log(logger)
+        assert logger.logger.level == LogVerbosity.from_string('INFO')
+
+    os.environ['JINA_LOG_LEVEL'] = 'SUCCESS'
```

Comment:
add a try catch mechanism that ensures thr os env is properly deleted but at same time captures the assertion error

Reply:
or split the test so that u can use fixtures to guarantee this

Reply:
Split the test and use fixtures in the latest commit. Thanks!